bin/yb-ctl

```diff
@@ -1357,6 +1382,9 @@ class ClusterControl:
         num_yb_admin_ts = None
 
         def call_check_for_ts(should_get_error):
+            if not call_check_for_processes(should_get_error):
+                raise ExitWithError("At least one master or tserver process is down.")
```

Comment:
I think we should also have one more debug function where, if we know some processes are down, we log out the respective yb-master.err / yb-tserver.err file for said process, in the hopes of highlighting why it crashed. We can then use that function both here and in line 1367, since at the end of that loop, we know we should be failing as well